mongodb/config.js

```diff
@@ -0,0 +1,16 @@
+// require('dotenv').load(); //loads .env vars
+var mongoose = require('mongoose');
+
+local: mongoose.connect('mongodb://localhost/test');
+//mongolab:  mongodb://<dbuser>:<dbpassword>@ds035503.mongolab.com:35503/mks-greenfield
```

Comment:
@SimonDing87 is there a reason the the keyword `local` is there? I'm just curious :) not entirely sure if it's necessary, just let me know.